src/app/shared/myimprovement-card/myimprovement-card.component.scss

```diff
@@ -0,0 +1,107 @@
+.improvements-card {
+    background-color: #fff;
+    border-radius: 20px;
+    box-shadow: 4px 4px 2.3px rgba(0, 0, 0, 0.05);
+    min-width: 320px;
+    min-height: auto;
+    max-width: 370px;
+    max-height: 220px;
+    margin: 15px;
+    font-family: Arial, sans-serif;
+    color: #333;
+    overflow: hidden;
+    }
+
+    .card-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 10px 15px;
+    border-radius: 20px 20px 0 0;
+    font-weight: bold;
+    font-size: 16px;
+    border: 1.82px solid #C0C0C0;
+    button{
+        padding: 5px 10px;
+        background-color: #fff;
+        border-radius: 20px;
+        display: flex;
+        mat-icon{
+            color: #572E91CC;
+        }
+    }
+    }
+    .card-header .improvement-title{
+        width: 70%;
+    }
+
+    .even-card {
+    .card-header{
+        color: #fff;
+        background-color: #9261D7;
+        mat-icon{
+            color: #9261D7;
+        }
+    }
+    }
+    .odd-card {
+        color: #000;
+        .card-header{
+            background-color:  #FF9911;
+            mat-icon{
+                color: #000;
+            }
+        }
+        }
+
+  .arrow-icon {
+    width: 30%;
+    font-size: 18px;
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .task-list {
+    list-style: none;
+    padding: 0;
+    font-size: 16px;
+    color: #555;
+    padding: 10px 15px;
+    font-weight: 400;
+  }
+
+  .task-list li {
+    margin: 5px 0;
+  }
+
+  .progress-bar {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0px 10px 10px 10px;
+  }
+
+  .progress-text {
+    font-size: 16px;
+    font-weight: 400;
+  }
+
+  .progress {
+    flex-grow: 1;
+    margin-left: 10px;
+    background: var(--Fills-Secondary, #78788029);
```

Comment:
--Fills-Secondary should be in lower case

Reply:
done